public/components/report_definitions/report_settings/report_settings_constants.tsx

```diff
@@ -5,6 +5,10 @@
 
 import { i18n } from '@osd/i18n';
 
+export const observabilityDashboardsPluginId = 'observabilityDashboards';
+
+export const notebooksReportSourceId = 'notebooksReportSource';
```

Comment:
Nitpick: to preserve style with the rest of the constants, these should be all upper-case

Reply:
Done [Updates constants names to SCREAMING_SNAKE_CASE](https://github.com/wazuh/wazuh-dashboard-reporting/pull/52/commits/e68e33c4c1b013b3e80ef99c17b0006156b92df6)